frame/democracy/src/benchmarking.rs

```diff
@@ -314,50 +309,31 @@ benchmarks! {
 	note_imminent_preimage {
 		// Num of bytes in encoded proposal
 		let b in 0 .. 16_384;
-		// Length of dispatch queue
-		let d in 0 .. 100;
 
-		let mut dispatch_queue = Vec::new();
 		// d + 1 to include the one we are testing
-		for i in 0 .. d + 1 {
-			let encoded_proposal = vec![0; i as usize];
-			let proposal_hash = T::Hashing::hash(&encoded_proposal[..]);
-			let block_number = T::BlockNumber::zero();
-			let referendum_index: ReferendumIndex = 0;
-			dispatch_queue.push((block_number, proposal_hash, referendum_index))
-		}
-		<DispatchQueue<T>>::put(dispatch_queue);
+		let encoded_proposal = vec![0; b as usize];
+		let proposal_hash = T::Hashing::hash(&encoded_proposal[..]);
+		let block_number = T::BlockNumber::one();
+		Preimages::<T>::insert(&proposal_hash, PreimageStatus::Missing(block_number));
 
 		let caller = funded_account::<T>("caller", b);
-		let encoded_proposal = vec![0; d as usize];
+		let encoded_proposal = vec![0; b as usize];
 	}: _(RawOrigin::Signed(caller), encoded_proposal)
 
 	reap_preimage {
 		// Num of bytes in encoded proposal
 		let b in 0 .. 16_384;
-		// Length of dispatch queue
-		let d in 0 .. 100;
-
-		let mut dispatch_queue = Vec::new();
-		for i in 0 .. d {
-			let encoded_proposal = vec![0; i as usize];
-			let proposal_hash = T::Hashing::hash(&encoded_proposal[..]);
-			let block_number = T::BlockNumber::zero();
-			let referendum_index: ReferendumIndex = 0;
-			dispatch_queue.push((block_number, proposal_hash, referendum_index))
-		}
-		<DispatchQueue<T>>::put(dispatch_queue);
 
-		let caller = funded_account::<T>("caller", d);
-		let encoded_proposal = vec![0; d as usize];
```

Comment:
^^^ note that was buggy. @marcio-diaz could you confirm this and recheck these benchmarks in case there are other such bugs?

Reply:
yep, shawn wrote `d` instead of `b`. i'll re-check the benchmarks.